src/components/Accounts/Actions/ActionModal.js

```diff
@@ -29,7 +29,7 @@ import {
 } from '../../../constants';
 
 import css from './PayWaive.css';
-import { localizeCurrencyAmount } from '../../util/localizeCurrencyAmount';
+import localizeCurrencyAmount from '../../util/localizeCurrencyAmount';
```

Comment:
after updating `src/hooks/index.js` file on the lines of other hooks exports (as suggested below in one of the comments), adjust this import